### ISSUE TITLE
"Not now" off the screen in portrait mode

### ISSUE THREAD

**Issue by jidanni** [brave/brave-browser#27825]
![Screenshot_20230115-175948.jpg](https://user-images.githubusercontent.com/1230959/212534651-294aa3bf-0e76-4d4d-8c55-e03e7f6d332b.jpg)
The user sees this prompt again and again, because in portrait orientation the  "not now" option is off the side of the screen. Yes this only affects some users who have larger font needs. Thus it is an accessibility problem.

**Comment by kjozwiak** [comment]
@jidanni thanks for the report/issue? Mind sharing some more information so we can try reproducing/fixing the above ASAP? Which version of `Android` are you on and what device are you using? Looks like the viewport is pretty small. Assuming one of the smaller devices?

**Comment by AlexeyBarabash** [comment]
Experiencing the same issue.
Device: `Samsung Note 8`
OS: `Android 8`
Browser version: `1.47.178`

STR:
1. Uninstall Brave Stable (if installed)
2. Install the latest Brave Stable from GP
3. Go through onboarding wizard 
See
![image](https://user-images.githubusercontent.com/24739341/213817511-90f683b5-daa3-4839-9afd-1172b7722b6e.png)

Additional not: I can't reproduce it on any other my Android device.

**Comment by jidanni** [comment]
The main thing you need to test, is in your Android settings, make the font size super large, and the screen zoom setting also super large. Yes it's for old people bad eyes. You will see the problem even even at just a little larger than normal. Android 12 Samsung A13 5G.

**Comment by AlexeyBarabash** [comment]
At https://github.com/brave/brave-browser/issues/27825#issuecomment-1399034545 zoom settings and text scale are default.

On my other phone I changed these settings to max values and the button is drawn well
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Description &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  | Screenshot
-------|-------------------
`Settings 1` | ![Screenshot_20230122_193111_Settings](https://user-images.githubusercontent.com/24739341/213930943-e48508cb-2586-45b8-be24-c343b0fc90b6.jpg)
`Settings 2` | ![Screenshot_20230122_193121_Settings](https://user-images.githubusercontent.com/24739341/213930906-e8853bdc-a800-4021-aba3-b7cb9499a768.jpg)
`Brave screen` | ![Screenshot_20230122_193040_Brave](https://user-images.githubusercontent.com/24739341/213930927-6ac72cf3-fcb2-4b20-87ae-772ef008a9dd.jpg)

**Comment by jidanni** [comment]
I remember I already successfully hit it in Landscape mode, so can no longer reproduce without reinstalling the app, etc.
So I hope other people can work on testing this bug.

**Comment by SergeyZhukovsky** [comment]
I didn't manage to replicate the Not Now button is off, but I remade that place and fixed the Add Widget button.

**Comment by SergeyZhukovsky** [comment]
We will open another issue if face a problem with a Now now button.

**Comment by kjozwiak** [comment]
The above requires `1.47.190` or higher for `1.47.x` verification 👍

**Comment by kjozwiak** [comment]
Moving this into `1.48.x` as we neve received another `C109` minor chromium bump that would have triggered another `1.47.x` release. We'll leave https://github.com/brave/brave-core/milestone/267 as there's code that landed in `1.47.x` but we'll close off the `1.47.x - Release #6` milestone and move all the issues within `1.48.x`.

The above can be verified with `1.48.156` for `1.48.x` as it's been merged into a `1.48.x` a while ago as per https://github.com/brave/brave-core/pull/16926.

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Pixel 6 Pro` running `Android 8` via the `Android Studio Emulator` using the following build(s):

```
Brave | 1.48.155 Chromium: 110.0.5481.77 (Official Build) (32-bit)
--- | ---
Revision | 65ed616c6e8ee3fe0ad64fe83796c020644d42af-refs/branch-heads/5481@{#839}
OS | Android 8.0.0; Build/OSR1.180418.026
```

* ensured that tapping on `Add Widget` via the onborading worked without any issues
* ensured that tapping on `Not now` correctly dismissed the widget onboarding without any issues

**`Example`** | **`Example`**
----------------|----------------
![image](https://user-images.githubusercontent.com/2602313/216905121-945a91ce-ed8f-4e8d-b524-2ceb725f8302.png) | ![image](https://user-images.githubusercontent.com/2602313/216906163-38b6442e-5d44-4c64-b139-83d75dfb54af.png)

Verification **PASSED** on `Pixel 6` running `Android 13` using the following build(s):

```
Brave | 1.48.155 Chromium: 110.0.5481.77 (Official Build) (32-bit)
--- | ---
Revision | 65ed616c6e8ee3fe0ad64fe83796c020644d42af-refs/branch-heads/5481@{#839}
OS | Android 13; Build/TQ1A.230105.002
```

* ensured that tapping on `Add Widget` via the onborading worked without any issues
* ensured that tapping on `Not now` correctly dismissed the widget onboarding without any issues

**`Example`** | **`Example`**
----------------|----------------
![image](https://user-images.githubusercontent.com/2602313/216907064-c52dc0dd-8c39-451f-bcea-c79a96649c19.png) | ![image](https://user-images.githubusercontent.com/2602313/216907109-bbd728a6-af87-42b3-807d-f9a6d5e24125.png)
